[PE] Add missing doc for submitting promotion approval request documentation in PE view (#1798)
wso2/docs-choreo-dev: en/pe-docs/docs/devops/cd-pipelines/submit-and-manage-environment-promotion-approval-requests.md

# Submit and Manage Environment Promotion Approval Requests

> In Choreo, when an administrator or platform engineer [configures the Environment Promotion workflow to require approval](../../governance/workflows/configure-approvals-for-choreo-workflows.md), you must submit a request to obtain approval to promote the deployment to a critical environment.

Reviewer: ```suggestion
If environment promotion workflows are configured for your organization, you must request and receive approval before deploying to a critical environment.
```

Reviewer: `...for your organization`
Not sure if this is a organization wide configuration or pipeline specific one

> Upon submitting a promotion approval request, Choreo notifies all authorized assignees via email about the [review request](../../governance/approvals/review-workflow-approval-requests.md). When an authorized assignee approves or rejects the request, you will receive an email with details of the decision.

Reviewer: ```suggestion
When you submit a promotion approval request, Choreo notifies all authorized reviewers via email. Once an authorized reviewer approves or rejects the request, you will receive an email with the decision details. See [Review Workflow Approval Requests](../../governance/approvals/review-workflow-approval-requests.md) for more information on workflow approval request reviews.
```

## Request approval for environment promotion

### Prerequisites

> - You should have a component created, built, and deployed to the first environment configured in your CD pipeline.

Reviewer: ```suggestion
- Ensure you have a component created, built, and deployed to the first environment of your [CD pipeline](...url/goes/here).
```

> - Ensure that an approval workflow is configured for environment promotion.

Reviewer: ```suggestion
- An approval workflow configured for environment promotion.
```

> - You should promote the component to a critical environment, such as production.

Reviewer: ```suggestion
- Ensure you are promoting the component to a critical environment, such as production.
```

> - You must have required permissions to promote components or at least `WORKFLOW-MANAGEMENT >> Create approval requests` permission

Reviewer: ```suggestion
- Ensure you have required permissions to either promote the component or to create approval requests.
```

Author: we need to be specific about the permission here.

Reviewer: Then shall we have something like this:
Ensure you have required permissions to either promote the component or to create approval requests. Approval request creation requires `Create approval requests` permission under `WORKFLOW-MANAGEMENT`.

WDYT?

Author: looks good

To request approval to promote a component from one environment to another critical environment, follow these steps:

1. Sign in to the [Choreo Console](https://console.choreo.dev/).
2. In the Choreo Console top navigation menu, select the **Organization**, then the **Project** and finally the relevant **Component**.
3. From the left navigation, Click **DevOps** and then click **CD Pipelines**
4. Go to the **Development** card and click **Request to Promote**.

> 5. In the **Request Approval** pane that opens, enter your request details and click **Submit**. This creates a request and notifies all authorized assignees via email about the request.

Reviewer: ```suggestion
5. In the **Request Approval** pane, enter your request details and click **Submit**. This creates a request and notifies all authorized assignees via email about the request.
```
